src/binder/bind_node_visitor.cpp

```diff
@@ -569,14 +573,18 @@ void BindNodeVisitor::Visit(common::ManagedPointer<parser::ColumnValueExpression
   SqlNodeVisitor::Visit(expr);
 
   sherpa_->CheckDesiredType(expr.CastManagedPointerTo<parser::AbstractExpression>());
+
   // TODO(Ling): consider remove precondition check if the *_oid_ will never be initialized till binder
   //  That is, the object would not be initialized using ColumnValueExpression(database_oid, table_oid, column_oid)
   //  at this point
   if (expr->GetTableOid() == catalog::INVALID_TABLE_OID) {
     std::tuple<catalog::db_oid_t, catalog::table_oid_t, catalog::Schema> tuple;
     std::string table_name = expr->GetTableName();
     std::string col_name = expr->GetColumnName();
-
+    if (table_name.empty() && col_name.empty() && expr->GetColumnOid() != catalog::INVALID_COLUMN_OID) {
+      throw BINDER_EXCEPTION(fmt::format("ORDER BY position \"{}\" is not in select list", expr->GetColumnOid()),
```

Comment:
You have to cast `col_oid_t` correctly. This does not compile.

Reply:
@apavlo Dear Professor Pavlo. The unit test always fail during ci. I've tried about 10 times, it still could not succeed. And I promise that the unit test could succeed in my local pc with ubuntu system.